data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -150,15 +155,19 @@ def _create_score_data(
 
         # merge state + county with score
         score_county_state_merged = score_df.merge(
-            county_state_merged, on="GEOID", how="left"
+            county_state_merged,
+            on="GEOID",  # GEOID is the county ID
+            how="left",
         )
 
         # check if there are census cbgs without score
         logger.info("Removing CBG rows without score")
```

Comment:
nit: should we switch all references to CBGs to tracts?

Reply:
Yes inded